Python/import.c

```diff
@@ -1744,27 +1744,30 @@ PyImport_ReloadModule(PyObject *m)
 PyObject *
 PyImport_Import(PyObject *module_name)
 {
+    _Py_IDENTIFIER(__import__);
+    _Py_IDENTIFIER(__builtins__);
+
     PyThreadState *tstate = _PyThreadState_GET();
-    static PyObject *silly_list = NULL;
-    static PyObject *builtins_str = NULL;
-    static PyObject *import_str = NULL;
+    PyObject *from_list = NULL;
+    PyObject *builtins_str = NULL;
+    PyObject *import_str = NULL;
     PyObject *globals = NULL;
     PyObject *import = NULL;
     PyObject *builtins = NULL;
     PyObject *r = NULL;
 
     /* Initialize constant string objects */
-    if (silly_list == NULL) {
-        import_str = PyUnicode_InternFromString("__import__");
-        if (import_str == NULL)
-            return NULL;
-        builtins_str = PyUnicode_InternFromString("__builtins__");
-        if (builtins_str == NULL)
-            return NULL;
-        silly_list = PyList_New(0);
-        if (silly_list == NULL)
-            return NULL;
-    }
+    import_str = _PyUnicode_FromId(&PyId___import__);
```

Comment:
You can define the variable here (same for the 2 other variables) and add a comment to mention that it's a borrowed ref

```suggestion
    PyObject *import_str = _PyUnicode_FromId(&PyId___import__);  // borrowed ref
```